src/main/java/calculator/NumberCalculator.java

```diff
@@ -0,0 +1,14 @@
+package calculator;
+
+import java.util.List;
+
+public class NumberCalculator {
+
+	public static final int DEFAULT_VALUE = 0;
+
+	public int calculate(List<Integer> numbers) {
+		return numbers
+			.stream()
+			.reduce(DEFAULT_VALUE, Integer::sum);
```

Comment:
Stream 활용 👍💯

조금 더 나아가서 아래와 같이 primitive type으로 변경한 뒤,  `sum` 함수를 사용하는 방식으로 변경해볼 수도 있을 것 같아요. 😃
```suggestion
        return numbers
            .stream()
            .mapToInt(Integer::intValue)
            .sum();
```